convert2rhel/pkghandler.py

```diff
@@ -458,6 +459,11 @@ def replace_non_rhel_installed_kernel(version):
 
     pkg = "kernel-%s" % version
 
+    utils.remove_pkgs(['kmod-kvdo', 'libreport'], critical=False)
+    for pkgname in set(system_info.pkg_blacklist):
```

Comment:
system_info.pkg_blacklist could have duplications?

Reply:
To be honest, I don't know.. but if has some dup, set will remove that.. I believe that's ok this approach.